enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -784,9 +784,11 @@
           [table (upload-test/create-upload-table!)]
           (let [db-id       (mt/id)
                 upload-csv! (fn []
-                              (upload-test/upload-example-csv! {:grant-permission? false
-                                                                :schema-name       (:schema table)
-                                                                :table-prefix      "uploaded_magic_"}))]
+                              (upload-test/do-with-uploaded-example-csv!
+                               {:grant-permission? false
+                                :schema-name       (:schema table)
+                                :table-prefix      "uploaded_magic_"}
+                               identity))]
```

Comment:
`upload-example-csv!` did not clean up after itself and created a bunch of `UPLOADED_MAGIC_EXAMPLE_CSV_FILE_` tables that never got cleaned up. Tests used to work ok because we deleted and recreated the `test-data` dataset several times in a test run. Now that we don't do that I had to fix these tests. Fixed by replacing this function with `do-with-uploaded-example-csv!` that properly cleans up after itself